service-activator/service-activator.bal

```diff
@@ -0,0 +1,35 @@
+import ballerina/http;
+
+type ClaimRequest record {|
+    string customerId;
+    string claimantName;
+    string policyNumber;
+    decimal claimAmount;
+    string accidentType;
+    string claimLocation;
+    string claimDate;
+|};
+
+type Claim record {|
+    *ClaimRequest;
+    string[] remarks;
+|};
+
+type ClaimHistory record {|
+    string customerId;
+    Claim[] claims;
+|};
+
+final http:Client firebaseClient = check new ("http://api.claimhistory.firebase.com.balmock.io");
+
+service /api/v1 on new http:Listener(8080) {
+    isolated resource function post claims/fruad(ClaimRequest claimRequest) returns boolean|error {
```

Comment:
having `fruad` api is bit wired, also word `fruad` is bit too strong. let's make it about `claims`.

Reply:
updated.